dockerfile

```diff
@@ -1,26 +1,26 @@
-# Use the official Node.js image as the base image
-FROM node:alpine
-
-# Set the working directory inside the container
+# ---- Base Node ----
+FROM node:alpine AS base
 WORKDIR /app
+COPY package.json .
 
-# Copy the package.json and package-lock.json (or yarn.lock) to the container
-COPY package.json ./
-
-# Set the NODE_ENV environment variable to "production"
-ENV NODE_ENV=production
-
-# Install the dependencies
+# ---- Dependencies ----
+FROM base AS dependencies
 RUN npm install
 
-# Copy the rest of the application files to the container
+# ---- Build ----
+FROM dependencies AS build
 COPY . .
-
-# Build the production version of the application
 RUN npm run build
 
-# Expose the port that your Next.js application will run on (by default Next.js uses port 3000)
+# ---- Release with Distroless ----
```

Comment:
oh I thought we would simply add postcss to prod dep instead of this

Reply:
nah, this fixes the issue too

Reply:
aight cool then